server/public/model/permission.go

```diff
@@ -252,6 +252,9 @@ var PermissionSysconsoleWriteEnvironmentPerformanceMonitoring *Permission
 var PermissionSysconsoleReadEnvironmentDeveloper *Permission
 var PermissionSysconsoleWriteEnvironmentDeveloper *Permission
 
+var PermissionSysconsoleReadEnvironmentMobileSecurity *Permission
+var PermissionSysconsoleWriteEnvironmentMobileSecurity *Permission
```

Comment:
I'm not familiar with the permission system but are these added by default to admins?

Reply:
I didn't find anywhere where we explicitly add these permissions for sys admins, but I understand sys admins always have all permissions, so it doesn't need to be explicit. Also, from my tests, I didn't need to do anything extra to have the sysadmin view this setting.

For other kind of admins (like read only admins) I guess that either this uses other higher level objects like the "SysconsoleReadPermissions" (which this new permission is already added) or that we wait for an admin to explicitly add these new permissions.

Reply:
Apparently we indeed have to do some sort of migration for permissions :sweat_smile: Added in one of the latest commits.